.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -21,12 +21,23 @@ additional information to help us prioritize the issue.
 -->
 
 
-## Positron Version:
+## Positron and language versions:
```

Comment:
I feel like we use "interpreter" a lot for this, e.g. <https://github.com/posit-dev/positron/wiki/Interpreter-Selection>. This isn't an ask for "language" vs. "interpreter" but just an observation that this will be good for us to develop a vocabulary guide for, moving forward.

Reply:
Let's change it!